includes/class-wcmp-frontend-settings.php

```diff
@@ -0,0 +1,343 @@
+            if (isset($this->settings['be_at_home_delivery_titel'])) {
+                return $this->settings['be_at_home_delivery_titel'];
+            }
+        }
+
+        /**
+         * @return mixed
+         */
+        public function pickup_titel() {
+            return $this->settings['pickup_titel'];
+        }
+
+        /**
+         * @return string
+         */
+        public function get_price_pickup() {
+            $price = $this->settings['pickup_fee'];
+            $total_price = $this->get_total_price_with_tax($price);
+            return $total_price;
+        }
+
+        /* Settings pickup express delivery */
+
+        /**
+         * @return bool
+         */
+        public function is_pickup_express_enabled() {
+            if(isset($this->settings['pickup_express_enabled'])) {
+                return (bool) $this->settings['pickup_express_enabled'];
+            }
+        }
+        /**
+         * @return mixed
+         */
+        public function pickup_express_titel() {
+            return $this->settings['pickup_express_titel'];
+        }
+
+        /**
+         * @return string
+         */
+        public function get_price_pickup_express() {
+            $price = $this->settings['pickup_express_fee'];
+            $total_price = $this->get_total_price_with_tax($price);
+            return $total_price;
+        }
+
+
+        /* Settings Monday delivery */
+        /**
+         * @return int
+         */
+        public function is_monday_enabled() {
+            if (isset($this->settings['saturday_cutoff_enabled'])) {
+                return $this->settings['saturday_cutoff_enabled'] ? 1 : 0;
+            }
+        }
+
+        /**
+         * @return mixed
+         *
+         * Cut-off time for monday delivery
+         */
+        public function get_saturday_cutoff_time() {
+            return $this->settings['saturday_cutoff_time'];
+        }
+
+
+        /**
+         * @return mixed
+         */
+        public function get_cutoff_time() {
+            if (
+                date_i18n( 'w' ) == self::DAYS_SATURDAY &&
+                isset( $this->settings['saturday_cutoff_time'] )
+            ) {
+                return $this->settings['saturday_cutoff_time'];
+            }
+
+            return $this->settings['cutoff_time'];
+        }
+
+        /**
+         * @return mixed
+         */
+        public function get_dropoff_delay() {
+            return $this->settings['dropoff_delay'];
+        }
+
+        /**
+         * @return mixed
+         */
+        public function get_deliverydays_window() {
+            return $this->settings['deliverydays_window'];
+        }
+
+        /**
+         * @return string
+         */
+        public function get_dropoff_days() {
+            return implode( ";", $this->settings['dropoff_days'] );
+        }
+
+        /**
+         * @return string
+         */
+        public function get_api_url() {
```

Comment:
Waar gebruik je `get_api_url()`? Normaal gesproken gebruik je geen getter bij een `cont` (omdat die altijd constant is). Om die andere plek kan je `WooCommerce_MyParcel_Frontend_Settings::BASE_URL` gebruiken

Reply:
Done!